data/sql/updates/pending_db_world/rev_1606590908424148300.sql

```diff
@@ -0,0 +1,62 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1606590908424148300');
+
+/*
+ * Update by Silker | github.com/Si1ker
+ * Copyright (C) | www.azerothcore.org
+*/
+
+/*
+References:
+https://www.youtube.com/watch?v=FkKoxtykUn0
+https://www.youtube.com/watch?v=QHWpZexZODY
+*/
+
+-- Update level range for Val'kyr Battle-maiden (59-60)
+UPDATE `creature_template` SET `maxlevel`=60 WHERE `entry`=31095;
+
+-- Update Terrifying Abomination wander distance
+UPDATE `creature` SET `wander_distance`=1 WHERE `guid` IN 
+(130505, 130506, 130507, 130508, 130511, 130512, 130513, 130515, 130516, 130517, 130519, 130521, 130522, 130524, 130525, 130526, 130527, 130528, 130530, 130531, 130532, 130535, 130536, 130537, 130539);
+
+-- Update level range for Terrifying Abomination (57-58)
+UPDATE `creature_template` SET `minlevel`=57 WHERE `entry`=31098;
+
+-- Fix Terrifying Abomination's SAI
+DELETE FROM `smart_scripts` WHERE `entryorguid`=31098 AND `source_type`=0;
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES 
+(31098, 0, 0, 0, 9, 0, 50, 1, 8, 20, 0, 0, 0, 11, 50335, 1, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 0, 'Terrifying Abomination - Within 8-20 Range - Cast Scourge Hook'),
+(31098, 0, 1, 0, 0, 0, 80, 0, 0, 0, 7000, 9000, 0, 11, 15496, 1, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Terrifying Abomination - In Combat - Cast Cleave');
+
+-- Update SAI, unit_flag and update level range for Knight of the Ebon Blade (57-58)
+UPDATE `creature_template` SET `minlevel`=57, `unit_flags`=557056, `AIName`='SmartAI' WHERE `entry`=31094;
```

Comment:
change the flags in this way
https://www.azerothcore.org/wiki/Dealing-with-SQL-files

Reply:
check changes proposed section